### PR TITLE
Update to Decidim 0.27

### PR DESCRIPTION
**Pull request by laurajaime**
This is a `clean-app` Decidim application.
Upgrade to 0.27: https://github.com/decidim/decidim/releases/tag/v0.27.0

<strike>
- [ ] Pending to review and merge: https://github.com/gencat/decidim-module-challenges/pull/68
</strike>

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -52,7 +52,8 @@
   # config.cache_store = :mem_cache_store
 
   # Use a real queuing backend for Active Job (and separate queues per environment)
-  config.active_job.queue_adapter = :delayed_job
+  # config.active_job.queue_adapter = :delayed_job
+  config.active_job.queue_adapter = :sidekiq
```
Comment (tramuntanal) [inline config/environments/production.rb]: ??

File: config/schedule.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
-every 1.day, at: "2:00 am" do # 1.minute 1.day 1.week 1.month 1.year is also supported
+env :PATH, ENV.fetch("PATH", nil)
+
+every 1.day, at: "5:00 am" do
   rake "decidim:metrics:all"
```
Comment (tramuntanal) [inline config/schedule.rb]: only `decidim:metrics:all` and `decidim:open_data:export` cronjobs for 0.27?
I think that at least we also need `decidim:delete_data_portability_files`. As we deploy in a server, `tmp:clear` will also be good every night.

File: public/sw.js.br
Comment (tramuntanal) [inline public/sw.js.br:1]: Should not be in `.gitignore`?

File: public/sw.js.gz
Comment (tramuntanal) [inline public/sw.js.gz:1]: Should not be in `.gitignore`?

File: public/sw.js.map.br
Comment (tramuntanal) [inline public/sw.js.map.br:1]: Should not be in `.gitignore`?

File: public/sw.js.map.gz
Comment (tramuntanal) [inline public/sw.js.map.gz:1]: Should not be in `.gitignore`?

**Review by tramuntanal — CHANGES_REQUESTED** [review]
Some small things to review please

**Comment by laurajaime** [comment]
@tramuntanal the chrome version in github actions has been updated, but Decidim in version 0.27 is not updated for this version. Since the chrome driver cannot be updated, the tests fail.

It seems that they will soon make a backport to 0.27 (https://github.com/decidim/decidim/pull/12351#issuecomment-1919154131), meanwhile we can merge when you do approve. The tests have passed locally.

File: Gemfile [hunk 3]
```diff
@@ -14,13 +14,17 @@ gem "decidim-sortitions", DECIDIM_VERSION
 
 gem "decidim-action_delegator", git: "https://github.com/CodiTramuntana/decidim-module-action_delegator"
 gem "decidim-budgets_paper_ballots", git: "https://github.com/digidemlab/decidim-module-budgets_paper_ballots"
-gem "decidim-challenges", git: "https://github.com/gencat/decidim-module-challenges", tag: "v0.2.0"
-gem "decidim-decidim_awesome", "~> 0.8.1"
-gem "decidim-file_authorization_handler", git: "https://github.com/CodiTramuntana/decidim-file_authorization_handler.git", tag: "v0.26.8.8"
+gem "decidim-cdtb", git: "https://github.com/CodiTramuntana/decidim-module-cdtb.git", branch: "main"
+gem "decidim-challenges", git: "https://github.com/gencat/decidim-module-challenges", branch: "bump/ruby_version"
```
Comment (tramuntanal) [inline Gemfile]: incorrect branch

**Review by tramuntanal — COMMENTED** [review]
Ok @laurajaime , let's do it like you say.

but first, challenges module branch in the Gemfile should be corrected

File: Gemfile (hunk 3 above)
Comment (laurajaime) [reply Gemfile]: done

**Review by tramuntanal — APPROVED** [review]

**Pull request merged**